Nullpointer Exception for /elements/geometry

The following message was visible in the logs upon an /elements/geometry request:

`Mar 01 10:55:02 heigitsv02 ignite.sh[9206]: Caused by: java.lang.NullPointerException`
`Mar 01 10:55:02 heigitsv02 ignite.sh[9206]: at org.heigit.ohsome.ohsomeapi.executor.ExecutionUtils.createOSMFeature(ExecutionUtils.java:387)`

using the following parameters:

```
bboxes: 7.109777450561525,46.00614082882865,7.187196731567385,46.04559279996195
filter: highway=track and (type:way)
time: 2007-11-01T00:00:00Z/2018-11-01T00:00:00Z/P1M
properties: tags,metadata
```

The issue is most likely caused by a given null as parameter instead of an empty EnumSet here:
https://github.com/GIScience/ohsome-api/blob/7a476ce54432bbe8cd7fc6429faa7dd487914251/src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ElementsRequestExecutor.java#L128

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,7 +8,7 @@ Changelog
 * fix some invalid filters in the default swagger examples ([#111])
 * fix returning invalid GeoJSON using empty coordinates for deletion contributions ([#129], [#131])
 * fix using a proper boolean data type instead of a string for contributionType in response ([#135])
-* fix NPE with createOSMFeature ([#141])
+* fix null pointer exception with createOSMFeature ([#141], [#145])
 * make sure geometry filters are applied to all returned features of elementsFullHistory requests ([#109])
 
 ### Performance and Code Quality
@@ -37,6 +37,7 @@ Changelog
 [#135]: https://github.com/GIScience/ohsome-api/pull/135
 [#138]: https://github.com/GIScience/ohsome-api/issues/138
 [#141]: https://github.com/GIScience/ohsome-api/issues/141
+[#145]: https://github.com/GIScience/ohsome-api/pull/145
 
 
 ## 1.3.2

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,7 +8,7 @@ Changelog
 * fix some invalid filters in the default swagger examples ([#111])
 * fix returning invalid GeoJSON using empty coordinates for deletion contributions ([#129], [#131])
 * fix using a proper boolean data type instead of a string for contributionType in response ([#135])
-* fix null pointer exception with createOSMFeature ([#141], [#145])
+* fix a null pointer exception in data extraction requests ([#141], [#145])
 * make sure geometry filters are applied to all returned features of elementsFullHistory requests ([#109])
 
 ### Performance and Code Quality

--- a/src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java
+++ b/src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java
@@ -2,6 +2,7 @@ package org.heigit.ohsome.ohsomeapi.executor;
 
 import java.io.Serializable;
 import java.util.Collections;
+import java.util.EnumSet;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
@@ -173,9 +174,9 @@ public class DataExtractionTransformer implements Serializable {
     properties.put("@validTo", endTimestamp);
     boolean addToOutput = addEntityToOutput(entity, geom);
     if (addToOutput) {
-      return Collections.singletonList(
-          exeUtils.createOSMFeature(entity, geom, properties, keysInt, includeTags,
-              includeOSMMetadata, isContributionsEndpoint, elementsGeometry, null));
+      return Collections.singletonList(exeUtils.createOSMFeature(entity, geom, properties,
+          keysInt, includeTags, includeOSMMetadata, isContributionsEndpoint, elementsGeometry,
+          EnumSet.noneOf(ContributionType.class)));
     } else {
       return Collections.emptyList();
     }

